### ISSUE TITLE
minor test data typo

### ISSUE THREAD

**Issue by wyn** [DEShawResearch/random123#5]
really minor point but I noticed that the test data in kat_vectors may be incorrectly copy-pasted?

lines 80 & 83, columns 9 & 10 (compare to line 77 columns 9 & 10).

Doesn't cause a problem as the expected values have been calculated with these inputs as is, 
but confused the hell out of me when I was trying to reproduce the test results (I'm in the process of porting Threefry code to OCaml).

**Comment by markmoraes** [comment]
yes, looks like it -- be5466cf34e90c6c got pasted into cols 8 and 9, rather than just 8, you're correct that it should be the same as line 77.  will fix in the next release, thanks.
